src/harness/os/linux.c

```diff
@@ -239,16 +262,16 @@ void resolve_full_path(char* path, const char* argv0) {
 }
 
 void OS_InstallSignalHandler(char* program_name) {
-
     resolve_full_path(_program_name, program_name);
 
     /* setup alternate stack */
     {
         stack_t ss = {};
-        /* malloc is usually used here, I'm not 100% sure my static allocation
-         is valid but it seems to work just fine. */
-        ss.ss_sp = (void*)alternate_stack;
-        ss.ss_size = SIGSTKSZ;
+        ss.ss_sp = malloc(2 * MINSIGSTKSZ);
```

Comment:
I switched to `malloc` because of the following error:
```
error: variably modified ‘alternate_stack’ at file scope
  249 | static uint8_t alternate_stack[SIGSTKSZ];
      |                ^~~~~~~~~~~~~~~
```

I had to (at least) double the stack because a `SIGSEGV` was being thrown whilst running our signal handler.